src/dialogs/vkbasalt.py

```diff
@@ -228,13 +228,16 @@ def __change_edge_detection_type(self, widget, edge_detection_type):
     def __import_clut(self, *args):
         def set_path(_dialog, response, _file_dialog):
             if response == -3:
-                _file = _file_dialog.get_file()
+                global file_path
```

Comment:
what's the purpose of making this a global var?

Reply:
It's needed for this: https://github.com/TheEvilSkeleton/Bottles/blob/1f0a808270a7632198a21fac82a5a2a0450363c3/src/dialogs/vkbasalt.py#L192

Reply:
You should avoid using global as much as possible. Global variables make the debug process hard as you lose where/when a value was assigned.

Reply:
What do you suggest instead?